src/frontend/package.json

```diff
@@ -6,13 +6,12 @@
   "scripts": {
     "build": "tsc --noEmit && webpack",
     "build-dev": "yarn copy-iframe-resizer; yarn build",
-    "extract-translations": "formatjs extract \"./**/*.ts*\" --out-file i18n/frontend.json --id-interpolation-pattern '[sha512:contenthash:base64:6]' --format react-intl-transformer.js",
-    "generate-l10n-template": "yarn extract-translations && rip json2pot './i18n/frontend.json' -o i18n/frontend.pot",
-    "generate-translations": "rip po2json './translations/*.po' -o './translations' -m './i18n/**/*.json'",
+    "extract-translations": "formatjs extract \"./**/*.ts*\" --ignore ./node_modules --ignore ./**/*.d.ts --out-file i18n/frontend.json --id-interpolation-pattern '[sha512:contenthash:base64:6]' --format crowdin",
```

Comment:
```suggestion
    "extract-translations": "formatjs extract './**/*.ts*' --ignore ./node_modules --ignore ./**/*.d.ts --out-file i18n/frontend.json --format crowdin",
```

Since you define manually id into `defineMessage.s` methods, you do not have to use `-id-interpolation-pattern`.

Reply:
Yes but it is recommended to use it, I think we should keep it as a guard.